FapgansControleBot/Controllers/message_controller.py

```diff
@@ -17,7 +21,21 @@ class MessageController:
     def __init__(self, unit_of_work: IUnitOfWork):
         self.unit_of_work = unit_of_work
 
-    def handle_message(self, update: Update, context: CallbackContext):
+    def handle_sticker(self, update: Update, context: CallbackContext):
```

Comment:
Nice, maybe rename this file to `StickerController` as well?

Reply:
Done